backend-server/lib.gradle

```diff
@@ -1,151 +1,152 @@
-                    "org.apache.pdfbox:jbig2-imageio:3.0.3"
-            ],
-            'nimbus'                       : "com.nimbusds:nimbus-jose-jwt:${nimbus_version}",
-            'hutool-core'                  : "cn.hutool:hutool-core:${hutool_version}",
-            'hutool'                       : [
-                    "cn.hutool:hutool-core:${hutool_version}",
-                    "cn.hutool:hutool-json:${hutool_version}",
-                    "cn.hutool:hutool-crypto:${hutool_version}",
-                    "cn.hutool:hutool-cache:${hutool_version}",
-                    "cn.hutool:hutool-http:${hutool_version}",
-            ],
-            'java-semver'                  : "com.github.zafarkhaja:java-semver:${java_semver_verion}",
-            'sentry'                       : [
-                    "io.sentry:sentry-spring-boot-starter:${sentry_version}",
-                    "io.sentry:sentry-logback:${sentry_version}",
-                    "io.sentry:sentry-apache-http-client-5:${sentry_version}",
-            ],
-            'minio'                        : "io.minio:minio:${minio_version}",
-            'v-client'                     : "cn.vika:vika-client:${v_java_sdk_version}",
-            'social-feishu'                : "cn.vika:vika-social-feishu:${v_social_version}",
-            'social-dingtalk'              : "cn.vika:vika-social-dingtalk:${v_social_version}",
-            'social-wecom'                 : "cn.vika:vika-social-wecom:${v_social_version}",
-            'social-qq'                    : "cn.vika:vika-social-qq:${v_social_version}",
-            'grpc-spring-boot'             : "net.devh:grpc-client-spring-boot-starter:${grpc_starter_version}",
-            'grpc-server-spring-boot'      : "net.devh:grpc-server-spring-boot-starter:${grpc_starter_version}",
-            'protobuf-java'                : "com.google.protobuf:protobuf-java:${protobuf_java_version}",
-            'io-grpc'                      : [
-                    "io.grpc:grpc-netty-shaded",
-                    "io.grpc:grpc-protobuf",
-                    "io.grpc:grpc-stub",
-            ],
-            'junit5'                       : 'org.junit.jupiter:junit-jupiter',
-            'assertj'                      : 'org.assertj:assertj-core',
-            'hamcrest'                     : 'org.hamcrest:hamcrest',
-            'jsonassert'                   : 'org.skyscreamer:jsonassert',
-            // Spring retry
-            'spring-retry'                 : "org.springframework.retry:spring-retry",
-            'awaitility'                   : "org.awaitility:awaitility",
-            'stripe'                       : "com.stripe:stripe-java:${stripe_version}",
-            'shedlock'                     : "net.javacrumbs.shedlock:shedlock-spring:${shedlock_version}",
-            'shedlock-redis'               : "net.javacrumbs.shedlock:shedlock-provider-redis-spring:${shedlock_version}",
-    ]
+  dependencies = [
```

Comment:
> It seems that the code style you use is different from us. your IDE tool has changed the indentation of this file, please use our code style file to format it? this configuration file is placed in the directory named: `apitable/backend-server/config/checkstyle/checkstyle.xml`. Thanks

Sorry,but this might not be my wrong.The others *.gradle files have the same Tab Size like my local.
And I have checked this file's history. It seems like the TAB Size or Indent change in this version below.

https://github.com/apitable/apitable/blob/256b416b2c243f8e18d0b7dc1680879e29e89a8c/backend-server/lib.gradle

Only this [backend-server/lib.gradle](https://github.com/apitable/apitable/pull/500/files/94aa2a710dcdfa7699aea4c5d9e4a8f3a6e648de#diff-c9a3be93a00d562bd48e0bab34a829f11b5d7929a2bebbee4abc9a0123cb99a8)
format is different from the others *.gradle.  
Shall we take a check?

Reply:
Are you using Intellij IDEA for coding？show me your groovy code style config under `Settings/Editor/Code Style/Groovy`, like below
![image](https://user-images.githubusercontent.com/39178385/223308260-d7d9ed6a-a6aa-48d8-a86b-35540f01ae7c.png)

Reply:
Yes I am using Intellij IDEA. This is my config below.
![20230307105837](https://user-images.githubusercontent.com/21049427/223309154-9bff5859-91ac-4c68-86f6-c32525fb102d.png)

Reply:
thanks for your help, I have fixed this IDE Setting!